lib/src/client/http2_connection.dart

```diff
@@ -101,7 +114,9 @@ class Http2ClientConnection implements connection.ClientConnection {
   Future<ClientTransportConnection> connectTransport() async {
     final Socket socket = await _createSocket();
     // Don't wait for io buffers to fill up before sending requests.
-    socket.setOption(SocketOption.tcpNoDelay, true);
+    if (!unix) {
```

Comment:
I think you can just do `socket.address.type != InternetAddressType.unix` here if you want to check that it's not a UDS.

Reply:
Good tips! Thanks very much.